docs/hashing.rst

```diff
@@ -10,7 +10,7 @@ Hash Method Generation
    Leave it at ``None`` which means that ``attrs`` will do the right thing for you, depending on the other parameters:
 
    - If you want to make objects hashable by value: use ``@attr.s(frozen=True)``.
-   - If you want hashing and comparison by object identity: use ``@attr.s(cmp=False)``
+   - If you want hashing and comparison by object identity: use ``@attr.s(eq=False)``
 
```

Comment:
I didn't execute this to check but this doesn't give comparison anymore like cmp did, right? I.e. s/comparison/equality?

Reply:
Ah yeah totally.